internal/provider/resource_cloudflare_tunnel_route.go

```diff
@@ -76,7 +89,12 @@ func resourceCloudflareTunnelRouteCreate(ctx context.Context, d *schema.Resource
 		return diag.FromErr(fmt.Errorf("error creating Tunnel Route for Network %q: %w", d.Get("network").(string), err))
 	}
 
-	d.SetId(newTunnelRoute.Network)
+	if virtualNetworkID != "" {
+		// It's possible to create several routes with the same network but different virtual network ids.
+		d.SetId(fmt.Sprintf("%s/%s", newTunnelRoute.Network, virtualNetworkID))
```

Comment:
if you are doing this, you'll want to make `virtual_network_id` `ForceNew` otherwise, the underlying ID will change everytime the `virtual_network_id` updates and you'll have issues in `Update`/`Read` operations.

we should also use [`stringChecksum`](https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/provider/utils.go#L63-L69) to keep the ID value looking consistent with others.

Reply:
![image](https://user-images.githubusercontent.com/10233442/172832219-a84b701e-1a01-4f5e-88e6-7ab3c2670bfb.png)
Have not found a way to update vnet. Changed it for `ForceNew` and used `stringChecksum` (btw, do you know why we need to use it (except following project code style)?)

Reply:
We always had slashes in ID even before (because CIDR always contains slash) but were not using `stringChecksum`.. so what is the real purpose?